vertical-pod-autoscaler/pkg/admission-controller/Makefile

```
@@ -88,3 +88,7 @@ format:
 	test -z "$$(find . -path ./vendor -prune -type f -o -name '*.go' -exec gofmt -s -w {} + | tee /dev/stderr)"
 
 .PHONY: all build test-unit clean format release
+
+.PHONY: admission-flags
+admission-flags:
```

```suggestion
.PHONY: document-flags
document-flags:
```

Would it make sense to make the name of the Makefile job the same across all components?

---

yeah it does! lol I feel dumb